linera-service/src/linera.rs

```diff
@@ -1682,6 +1693,49 @@ async fn main() -> Result<(), anyhow::Error> {
             Ok(())
         }
 
+        ClientCommand::Net(net_command) => match net_command {
+            NetCommand::Up => {
+                let network = Network::Grpc;
+                let mut net = LocalNetwork::new(network, 1)?;
+                let client1 = net.make_client(network);
+                let client2 = net.make_client(network);
+
+                net.generate_initial_validator_config().await?;
+                client1.create_genesis_config().await?;
+                client2.wallet_init(&[]).await?;
+
+                // Create initial server and client config.
+                net.run().await?;
+                let net_path = net.net_path();
+
+                eprintln!(
+                    "\nLinera net directory available at: {}",
+                    net_path.display()
+                );
+                eprintln!("To configure your Linera client for this network, run:\n");
+                eprintln!(
+                    "{}",
+                    format!(
+                        "export LINERA_WALLET=\"{}\"",
+                        net_path.join("wallet_0.json").display()
+                    )
+                    .bold()
+                );
+                eprintln!(
```

Comment:
Weren't these two meant to go to stdout?

Reply:
I'm not sure tbh. There is no value in piping them anywhere really they are meant to be copy/pasted.